src/modules/filesystem/wrap_Filesystem.cpp

```diff
@@ -828,7 +834,10 @@ int extloader(lua_State *L)
 
 			Filesystem::Info info = {};
 			if (!inst->getInfo(element.c_str(), info) || info.type == Filesystem::FILETYPE_DIRECTORY)
+			{
+				errstr << "\n\t  tried '" << element << "' in save directory";
```

Comment:
I think this isn't guaranteed to only look (or work) in the save directory right now, especially if love 12's new mount APIs have been used.

Reply:
What would be a good replacement text for those?

Reply:
Maybe it could be something like this for the Lua loader:
"No Lua module 'path/to/module' in LOVE filesystem paths:"

And for the C module loader:
"No C module 'path/to/module' in LOVE filesystem paths:"

Does that make sense? I haven't thought about this too hard.

Reply:
For the C module paths, I was thinking of resolving the fullpath then printing the absolute path, but then I realize `PHYSFS_getRealDir` only works when the file is there (which most likely succeeded when this codepath is reached, defeating the purpose).